08LessonEight/homework/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Exercise 08</title>
+    <link rel="stylesheet" href="css\style.css">
```

Comment:
While it _works_ using the backslash `\`, always use a forward slash for html `/`.